java/automl/README.md

```diff
@@ -49,3 +49,23 @@ The following guides illustrate how to run the project:
 * curl -X POST "http://localhost:8080/test/insert" -H "accept: */*" -H "Content-Type: application/json" -d "{ \"id\":
   444, \"name\": \"333\"}"
 
+#### Build docker image
+
+build executable jar
```

Comment:
I think All mvn commands should be moved into Dockerfile so that users can use the Dockerfile only to build the image like the command I suggested in the comment of the Readme in the root folder